scripts/gen_client_cert.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
```

Comment:
Since this is near duplicated of gen_keys.sh, I would recommend making this a sourcable bit of bash code that can be used by gen_keys.sh.  See http://mywiki.wooledge.org/BashGuide/Sourcing